app/src/main/java/org/eyeseetea/malariacare/domain/boundary/IUserLocalDataSource.java

```diff
@@ -0,0 +1,9 @@
+package org.eyeseetea.malariacare.domain.boundary;
+
+import org.eyeseetea.malariacare.domain.entity.UserAccount;
+
+public interface IUserLocalDataSource {
```

Comment:
Abstractions should not be local or remote, only implementations. This interface should be IUserAccountRepository